### ISSUE TITLE
How to make custom requests using ocktokit (for istance add specific query)

### ISSUE THREAD

**Issue by fabriziogianni7** [octokit/request.js#575]
https://github.com/orgs/octokit/discussions/39
https://stackoverflow.com/questions/75813165/how-to-make-custom-requests-using-ocktokit-for-istance-add-specific-query

Hey guys I've been looking for this for a while but I don't understand how it works, please answer here on in stackoverflow if you want, here the description:

In my project, we use ocktokit to make requests to github. it's all good, except that I need to make custom queries on issues, for istance, I want to add:

`-label:bot:ReadyToSign -label:Application:AddressNotValid`

how should I do?
I tried using ockokit/request API and using something like that:

```
 let allGithubIssues = await  octokit.request(`GET /repos/{owner}/{repo}/issues?q=-label:bot:ReadyToSign`, {
      owner: "my-repo-owner",
      repo: "my-repo",
    });
```

but is not working.

I would like also to search for text into the comments or body of issues, something like

`in:body` or `in:comments`

**Comment by wolfy1339** [comment]
I'm not entirely sure how to do that kind of query using the requests module.

What you can do, is get all the issues for a repo, and then filter the data by looking through their tags and body

**Comment by fabriziogianni7** [comment]
Yeah, I'm doing that but it's kind of slow. Would be amazing if I can submit a custom query somehow
